support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -9,10 +9,7 @@ export const pageUrlRegexes = {
 	contributions: {
 		allLandingPagesAndThankyouPages:
 			'/checkout|one-time-checkout|contribute|thankyou|thank-you(/.*)?$',
-		notUkLandingPage: '/us|au|eu|int|nz|ca/contribute(/.*)?$',
```

Comment:
I think all these pre-date the 2-step checkout, and so the name "landing page" is now incorrect